packages/devtools/cli/README.md

```diff
@@ -4,11 +4,43 @@ DXOS command line interface.
 
 <!-- toc -->
 * [DX CLI](#dx-cli)
+* [Instalation](#instalation)
+* [Running an Agent](#running-an-agent)
 * [Development](#development)
 * [Usage](#usage)
-* [Table of contents](#table-of-contents)
 <!-- tocstop -->
 
+# Instalation
+```terminal
+npm install -g @dxos/cli@main
+```
+---
+# Running an Agent
+Agent is a name for DXOS Client which is run by CLI in the background in the daemonized process. 
+## Starting agent
+Agent is automatically started by each command that requires Client (to avoid this behavior use `--no-agent` flag). You can use `--profile` flag (default value is `default`) to run agent in an isolated profile, and `--foreground` to run agent in attached process.
```

Comment:
I would split out the possible flags onto separate lines. I can help rewrite this section to try and make it clearer about "commands that require Clients", etc.